documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,467 @@
+# Get Started
+
+<!--- ash-hq-hide-start --> <!--- -->
+
+This documentation is best viewed at [ash-hq.org](https://ash-hq.org)
+
+<!--- ash-hq-hide-stop --> <!--- -->
+
+## Goals
+
+In this guide we will:
+
+1. Create a new Phoenix application and add Ash, AshPhoenix and AshPostgres as dependencies
+2. Create a simple set of resources and show they can be used
+3. Illustrate some core concepts of Ash
+4. Point you to good next resources so you can explore Ash further
+
+## Things you may want to read first
+
+- [Install Elixir](https://elixir-lang.org/install.html)
+- [Phoenix - Up and Running Guide](https://hexdocs.pm/phoenix/up_and_running.html)
+- [Philosophy Guide](/documentation/tutorials/philosophy.md)
+
+## Requirements
+
+If you want to follow along yourself, you will need the following things:
+
+1. Elixir (1.12 or later) and Erlang (22 or later) installed
+2. PostgreSQL installed
+3. A text editor to make the changes that we make
+4. A terminal to run the examples
+
+## Steps
+
+For this tutorial, we'll use examples based around creating a help desk.
+
+We will make the following resources:
+
+- `PhoenixHelpdesk.Support.Ticket`
+- `PhoenixHelpdesk.Support.Representative`
+
+The actions we will be able to take on these resources include:
+
+- Opening a new Ticket
+- Closing a Ticket
+- Assigning a Ticket to a representative
+
+### Create a New Phoenix Project
+
+_This section is based the [Phoenix installation docs](https://hexdocs.pm/phoenix/installation.html). For more details go there._
+
+We first need to create a fresh phoenix project using the phoenix project generator.
```

Comment:
`Phoenix` should be uppercase

Reply:
did a ctrl+f for the whole document.
addressed: fb62b925997118b19a79678abf492252ec821c4d